tests/baselines/reference/arrayContextualTyping.symbols

```diff
@@ -0,0 +1,105 @@
+=== tests/cases/compiler/arrayContextualTyping.ts ===
+// Setup
+declare class MyCustomArray extends Array<number> {
+>MyCustomArray : Symbol(MyCustomArray, Decl(arrayContextualTyping.ts, 0, 0))
+>Array : Symbol(Array, Decl(lib.es5.d.ts, --, --), Decl(lib.es5.d.ts, --, --))
+
+    isCustom: true;
+>isCustom : Symbol(MyCustomArray.isCustom, Decl(arrayContextualTyping.ts, 1, 51))
+}

```

Comment:
@RyanCavanaugh Could you please clarify what is the purpose of this `isCustom` property? Looking at this, it seems to me like it is a condition to run a different set of tests, if so it does not seem like a good practice to have conditional tests.